x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/fields/create_field/required_parameters_forms/token_count_type.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { i18n } from '@kbn/i18n';
+
+import { AnalyzerParameter } from '../../../field_parameters';
+import { STANDARD } from '../../../../../constants';
+import { EditFieldSection } from '../../edit_field';
+
+export const TokenCountTypeRequiredParameters = () => {
```

Comment:
Sure, I will take a look.

Oops! I did not test selecting the `language` option. I did not want the select to span the full width of the screen, but this will probably be obsolete if I add the description text to the left.

Reply:
fixed